build-system/test-configs/forbidden-terms.js

```diff
@@ -1338,10 +1323,9 @@ function matchForbiddenTerms(srcFile, contents, terms) {
       // NOTE: we could do a glob test instead of exact check in the future
       // if needed but that might be too permissive.
       if (
-        isInBuildSystemFixtureFolder(srcFile) ||
```

Comment:
`build-system/babel-plugins/**/fixtures/**/*.js` is already listed on `.eslintignore`